pyproject.toml

```diff
@@ -23,6 +23,8 @@ packaging = "^21.3"
 colorama = {version = "^0.4.3", optional = true}
 # For Pydantic
 dataclasses = {version = "^0.7", python = "~3.6"}
+redis = "*"
```

Comment:
1. Is there a known minimum version of `redis` that's required?
2. Can we make this into an optional/extras dependency, i.e. so that `pip install diffsync` doesn't pull in redis (and only supports local storage as a result), but `pip install diffsync[redis]` does?

Reply:
good idea